Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM openjdk:17-oracle
+FROM openjdk:21-oracle
```

Comment:
OpenJDK is deprecated. See here: https://hub.docker.com/_/openjdk
Maybe go for `eclipse-temurin:21`

Reply:
Have updated this in new push